SwiftRustIntegrationTestRunner/integration-test-create-swift-package/src/main.rs

```diff
@@ -8,7 +8,7 @@ fn main() {
     create_package(CreatePackageConfig {
         bridge_dir: PathBuf::from("swift-package-rust-library-fixture/generated"),
         paths: HashMap::from([
-            (Platform::MacOS, PathBuf::from("swift-package-rust-library-fixture/target/x86_64-apple-darwin/debug/libtest_swift_packages.a") as _),
+            (Platform::MacOS, PathBuf::from("swift-package-rust-library-fixture/target/universal/libtest_swift_packages.a") as _),
```

Comment:
Hmm..

Wouldn't this change just be testing that universal builds work. 

Which would mean that we're adding an extra build in CI and adding that `lipo` call in order to just test that `lipo` actually works?

If so, maybe we just tweak the book to demonstrate a universal build and call it a day?

Reply:
I actually did this because I wanted `./test-integration.sh` to run on my m1 mac. I messed about having the `integration-test-create-swift-package` subcrate use the `TARGET` triple that it's compiled o and then have the `build.sh` do `if [ $(uname -m) = "arm64" ]; then ...` but found this to be worse.

I concluded that just making the CI library just be universal solved this problem. I'm fine with either option. 

When I did #55, I had to change `x86_64` to `aarch64` in two places (and then not commit them) to verify the changes were correct before submitting the PR.

Reply:
Ah got it thanks for explaining.

Looks like a clean debug build of the fixture on my machine takes about 9 seconds (mainly syn).

If we guesstimate that on the weaker CI hardware it takes like ... say 20 ... this change would add roughly 20 seconds to CI (back of napkin math).

So.. nothing that we need to worry about for now.

So this universal approach sounds good to me. 

If you don't mind tweaking the Swift Package section of the book we can land this CI tweak without changes.